### PR TITLE
docs(globals): enhance configuring globals docs

### PR DESCRIPTION
**Pull request by fzn0x**
### Description

Setting `vitest/globals` may not work smoothly without configuring the types root explicitly.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [x] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: docs/config/index.md [hunk 1]
```diff
@@ -382,6 +382,18 @@ To get TypeScript working with the global APIs, add `vitest/globals` to the `typ
 }
 ```
 
+Sometimes the configuration mentioned above might not work because your workspace cannot find the types for `vitest/globals`. If that's the case, you can add `typeRoots` to your `tsconfig.json` file. Afterward, try restarting your TypeScript server. For example in VSCode, press `F1` and select `TypeScript: Restart TS server`
```
Comment (AriPerkkio) [inline docs/config/index.md:385]: I'm not sure if it's Vitest's documentation's place to instruct restarting IDEs Typescript servers. It's IDE specific.

File: docs/config/index.md [hunk 2]
```diff
@@ -382,6 +382,18 @@ To get TypeScript working with the global APIs, add `vitest/globals` to the `typ
...
+
+```json
+// tsconfig.json
+{
+  "compilerOptions": {
+    "types": ["vitest/globals"],
+    "typeRoots": ["./node_modules/@types/", "./types", "./node_modules"]
```
Comment (AriPerkkio) [inline docs/config/index.md:392]: In what cases doesn't the default `typeRoots` work? Could you setup a minimal reproduction with https://vitest.new/ ?

File: docs/config/index.md (hunk 1 above)
Comment (fzn0x) [reply docs/config/index.md:385]: You are true @AriPerkkio , I think VSCode having a problem to recognize vitest/globals, adding that steps make it works but after for some times I can use vitest/globals without typeRoots and restarting VSCode TS server.

File: docs/config/index.md (hunk 2 above)
Comment (fzn0x) [reply docs/config/index.md:392]: I think it's IDE specific. @AriPerkkio

File: docs/config/index.md (hunk 1 above)
Comment (fzn0x) [reply docs/config/index.md:385]: Any suggestions regarding that?

Comment (AriPerkkio) [reply docs/config/index.md:385]: Is it really Vitest specific? Or is VSCode just not automatically restarting the Typescript server after `tsconfig.json` was modified?

**Comment by sheremet-va** [comment]
I don't think this is Vitest specific. We don't require any `typeRoot` to work, and an LSP restart is always a good thing to do after updating a `tsconfig`.

**Pull request closed without merging**